### ISSUE TITLE
patternlab:loadstarterkit fails with TypeError: Path must be a string. Received undefined with Node 8.9.1

### ISSUE THREAD

**Issue by ThePeach** [Comcast/patternlab-edition-node-webpack#3]
## I found the following issue:

I'm using Node lts, version 8.9.1
I've installed `starterkit-mustache-demo`.
Running the loadstarterkits command fails with the following output

```
$ npm run patternlab:loadstarterkit -- --kit=starterkit-mustache-demo

> edition-node-webpack@1.0.3 patternlab:loadstarterkit /home/peach/public_html/patternlab-edition-node-webpack
> node patternlab.js loadstarterkit "--kit=starterkit-mustache-demo"

Loading engines from the core...


Loading engines from the edition or test directory...

  mustache: good to go

Done loading engines.


====[ Pattern Lab / Node - v2.12.0 ]====

TypeError: Path must be a string. Received undefined
    at assertPath (path.js:28:11)
    at Object.join (path.js:1251:7)
    at loadStarterKit (/home/peach/public_html/patternlab-edition-node-webpack/node_modules/patternlab-node/core/lib/starterkit_manager.js:19:14)
    at Object.load_starterkit (/home/peach/public_html/patternlab-edition-node-webpack/node_modules/patternlab-node/core/lib/starterkit_manager.js:103:7)
    at loadStarterKit (/home/peach/public_html/patternlab-edition-node-webpack/node_modules/patternlab-node/core/lib/patternlab.js:286:24)
    at Object.loadstarterkit (/home/peach/public_html/patternlab-edition-node-webpack/node_modules/patternlab-node/core/lib/patternlab.js:646:7)
    at loadstarterkit (/home/peach/public_html/patternlab-edition-node-webpack/patternlab.js:43:14)
    at Object.<anonymous> (/home/peach/public_html/patternlab-edition-node-webpack/patternlab.js:94:7)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
```

With version 8.8.1 I have no problems at all

**Comment by ThePeach** [comment]
So the problem presents itself on version 6 as well, and upon further investigation turns out it's a missing configuration variable in `patternlab-config.json`:   `"starterkitSubDir": "dist"`

:+1:

**Comment by paintedbicycle** [comment]
I've updated to the newer version that is supposed to fix this (with the missing /dist/ configuration), but it hasn't for me. 

I installed with the following terminal commands:

```
npm install
npm run patternlab:build
npm run patternlab:loadstarterkit --kit=starterkit-mustache-demo
```

And I get the same error:

```
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:28:11)
    at Object.join (path.js:1251:7)
    at loadStarterKit (/Users/paintedbicycle/Sites/patternlab-edition-node-webpack/node_modules/patternlab-node/core/lib/starterkit_manager.js:19:14)
    at Object.load_starterkit (/Users/paintedbicycle/Sites/patternlab-edition-node-webpack/node_modules/patternlab-node/core/lib/starterkit_manager.js:103:7)
    at loadStarterKit (/Users/paintedbicycle/Sites/patternlab-edition-node-webpack/node_modules/patternlab-node/core/lib/patternlab.js:286:24)
    at Object.loadstarterkit (/Users/paintedbicycle/Sites/patternlab-edition-node-webpack/node_modules/patternlab-node/core/lib/patternlab.js:646:7)
    at loadstarterkit (/Users/paintedbicycle/Sites/patternlab-edition-node-webpack/patternlab.js:43:14)
    at Object.<anonymous> (/Users/paintedbicycle/Sites/patternlab-edition-node-webpack/patternlab.js:94:7)
    at Module._compile (module.js:612:30)
    at Object.Module._extensions..js (module.js:623:10)
```

**Comment by mbulfair** [comment]
@paintedbicycle What version of Node/NPM are you using?

**Comment by paintedbicycle** [comment]
@mbulfair 
Node 8.8.0
NPM 5.4.2

**Comment by paintedbicycle** [comment]
The undefined variable was not the path, but instead it was the npm variable.

#5 is my PR that should fix this

**Comment by mbulfair** [comment]
Issues should be resolved in the latest release [1.0.3.2](https://github.com/Comcast/patternlab-edition-node-webpack/releases/tag/v1.0.3.2)
